cmd/gocert/main.go

```diff
@@ -1,5 +1,36 @@
 package main
 
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	server "github.com/canonical/gocert/api"
+	"github.com/canonical/gocert/internal/certdb"
+)
+
 func main() {
-	// ListenAndServe
+	certPathPtr := flag.String("cert", "", "A path for a certificate file to be used by the webserver")
+	keyPathPtr := flag.String("key", "", "The path for a private key for the given certificate")
+	dbPathPtr := flag.String("db", "./certs.db", "The path of the SQLite database for the repository")
+	flag.Parse()
+
+	if *certPathPtr == "" || *keyPathPtr == "" {
+		fmt.Fprintf(os.Stderr, "Providing a certificate and matching private key is extremely highly recommended. To do so, run: --cert <path/to/certificate> --key <path/to/key>. Using self signed certificates.")
+	}
+	_, err := certdb.NewCertificateRequestsRepository(*dbPathPtr, "CertificateRequests")
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Couldn't connect to database: %s", err)
+		os.Exit(1)
```

Comment:
For this construct, we could use the `log` module:
```suggestion
		logger.Fatalf(os.Stderr, "Couldn't connect to database: %s", err)
```

Reply:
Done. The logger doesn't support specifying the output writer in the `Fatalf` call itself, so I'm just setting the output to stderr in the beginning for now. This can have more advanced logic later.